### PR TITLE
Fix container curl opensuse.org issue

### PR DESCRIPTION
**Pull request by pdostal**
- Related ticket: [poo#167030](https://progress.opensuse.org/issues/167030)
- Verification run: 
sle-15-SP3-EC2-BYOS-Updates.aarch64	[publiccloud_containers@64bit](https://pdostal-server.suse.cz/tests/7678)
sle-15-SP6-Azure-BYOS-Updates.aarch64	[publiccloud_containers@64bit](https://pdostal-server.suse.cz/tests/7677)
sle-12-SP5-GCE-Updates.x86_64 [publiccloud_containers@64bit](https://pdostal-server.suse.cz/tests/7679)

### CODE REVIEW

File: tests/containers/container_engine.pm [hunk 1]
```diff
@@ -111,7 +111,7 @@ sub basic_container_tests {
     assert_script_run("$runtime image rm example.com/tw-commit_test");
 
     ## Test connectivity inside the container
-    assert_script_run("$runtime container exec basic_test_container curl -sfI https://opensuse.org", fail_message => "cannot reach opensuse.org");
+    script_retry("$runtime container exec basic_test_container curl -sfIL https://opensuse.org", retry => 3, delay => 5, fail_message => "cannot reach opensuse.org");
```
Comment (mloviska) [inline tests/containers/container_engine.pm:114]: it would be better to use `conncheck.opensuse.org` as it was created for this purpose

Comment (asmorodskyi) [inline tests/containers/container_engine.pm:114]: I would put much bigger delay , let say 60 then we will increase our chances ...

Comment (grisu48) [reply tests/containers/container_engine.pm:114]: I would also increase the delay to 60. If it fails now, give the network some time before trying again

Comment (pdostal) [reply tests/containers/container_engine.pm]: It's not timeout, timeout is 90. This is just delay and is not much important. The important part is `retry => 3`.

**Review by ricardobranco777 — APPROVED** [review]

**Review by mloviska — APPROVED** [review]

**Pull request merged**